src/AutoBuild.jl

```diff
@@ -226,9 +226,8 @@ function build_tarballs(ARGS, src_name, src_version, sources, script,
             @info("Committing and pushing $(src_name)_jll.jl wrapper code version $(build_version)...")
         end
 
-        # For deploy discard build-only dependencies and transform in PackageSpec
+        # For deploy discard build-only dependencies
         filter!(dep -> !isa(dep, BuildDependency), dependencies)
-        dependencies = getpkg.(dependencies)
```

Comment:
Yesterday I restricted the signature to `Array{Dependency}` in #656 because that's what we need in Yggdrasil and I wanted to preemptively block `BuildDependency`s.  It is great if we don't need `PackageSpec`'s further down here.  Unfortunately, this is a code path difficult to test (and hence easy to break) because we'd need to publish something to a GitHub repository.

Reply:
If that helps: I've tested it by publishing it to my github repo: https://github.com/laborg/hidapi_jll.jl

Reply:
Great to know!